src/RestrictedTokenFaucet.sol

```diff
@@ -0,0 +1,59 @@
+pragma solidity >=0.5.0;
+
+interface ERC20Like {
+    function balanceOf(address) external view returns (uint256);
+    function transfer(address,uint256) external; // return bool?
```

Comment:
DSToken.transfer returns a bool, so I think for clarity it's better to be aligned to its implementation (even though it doesn't make any difference in terms of function selector).

```suggestion
    function transfer(address,uint256) external returns (bool);
```

Reply:
I don't think this works here since we are working with more interfaces than just DSToken and can't always expect well behaved transfers